### PR TITLE
modifications to calendar display

### PR DESCRIPTION
**Pull request by jasmine2000**
timeblock display:
- if a user has no availabilities, don't show the calendar.
- if a user has no availabilities but they are editing their own calendar, show the calendar.
- if a timeblock as passed, don't show it on the calendar.
- show request button even if user does not have access but styled to look disabled

calendar display:
- cannot click edit if you don't have a meeting link
- past days are grey, current day is blue, future days are white

requesting:
- after clicking request, message directs you to meetings tab

other minor styling/linguistic changes also 👯

### CODE REVIEW

File: client/components/Profile/CalendarComponent.vue [hunk 1]
```diff
@@ -301,7 +334,7 @@ export default {
                 headers: {'Content-Type': 'application/json'},
                 credentials: 'same-origin',
                 body: JSON.stringify({
-                start: date.toString(),
+                  start: date.toString(),
```
Comment (hxliu33) [inline client/components/Profile/CalendarComponent.vue:337]: lol wut why did we indent

File: client/components/Profile/CalendarComponent.vue [hunk 2]
```diff
@@ -335,6 +368,9 @@ export default {
                 if (!r.ok) {
                     throw new Error(res.error);
                 }
+                const message = 'View requested meeting in Meetings tab!';
+                this.$set(this.alerts, message, 'success');
+                setTimeout(() => this.$delete(this.alerts, message), 3000);
```
Comment (hxliu33) [inline client/components/Profile/CalendarComponent.vue:373]: ooh i like, good success affirmative message

File: client/store.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ const store = new Vuex.Store({
     user: null, // logged in user
     userId: null, // User ID of logged in user
     users: [], // All users
-    hasAccess: false,
+    hasAccess: null,
```
Comment (hxliu33) [inline client/store.ts:17]: is there a reason we changed default to `null` from `false`?

**Review by hxliu33 — APPROVED** [review]

File: client/store.ts (hunk 3 above)
Comment (jasmine2000) [reply client/store.ts:17]: the behavior will be the same (stuff checks if hasAccess==null) and it seemed more intuitive for the uninitialized value to be null? if u prefer false i can

**Pull request merged**